src/Symfony/Component/Notifier/Bridge/Octopush/OctopushTransport.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Octopush;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class OctopushTransport extends AbstractTransport
+{
+    protected const HOST = 'www.octopush-dm.com';
+
+    private $userLogin;
+    private $apiKey;
+    private $smsSender;
+    private $smsType;
+
+    public function __construct(string $userLogin, string $apiKey, string $from, string $smsType, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->userLogin = $userLogin;
+        $this->apiKey = $apiKey;
+        $this->smsSender = $from;
+        $this->smsType = $smsType;
```

Comment:
```suggestion
        $this->sender = $from;
        $this->type = $smsType;
```

As far as I can see, only SMS is supported, so no need for a `sms` prefix 🤔

Reply:
At Octopush there are "types" of SMS. The API requires this option. I can force the use of the default "world" option but users would no longer be able to specify a type. In some cases, this option impacts the cost of the SMS. You can see the details of the required fields: https://www.octopush.com/api-sms-documentation/parametres/

Reply:
Oh sorry for being not clear I just mean we can rename the variable to type instead of smsType...

Reply:
Ah I got it, thank you.